src/components/command-bar/commands/search/helpers/hit-counts-provider.tsx

```diff
@@ -0,0 +1,30 @@
+import { SearchableContentType } from 'contexts'
+import { ReactNode, createContext, useContext, useState } from 'react'
+
+const SearchHitsContext = createContext([])
```

Comment:
❓ we have stored other contexts in `/src/contexts`, I'm not super opinionated but just pointing it out in case it feels better for all of our providers to live in one place.

Reply:
Ah yeah great call I forgot about that!

I think in this case the provider feels component-specific, say similar to the tab nesting styles provider, or tab groups provider. I think a single home for _all_ contexts could get unwieldy... where a provider is only used by a component and its partials, I think it could make sense to keep it co-located with whatever scope it's intended to apply to (sometimes `component`-specific, sometimes `view`-specific, etc). Or to put it another way, I think `src/contexts` could make a lot of sense for any contexts we use in `_app` 🤔